gce/load-test.sh

```diff
@@ -0,0 +1,47 @@
+#!/bin/bash
+
+set -o nounset
+set -o pipefail
+set -o xtrace
+
+# When running in prow, the working directory is the root of the test-infra
+# repository.
+
+# Taint the Linux nodes to prevent the test workloads from landing on them.
+# TODO: remove this once the issue is resolved:
+# https://github.com/kubernetes/kubernetes/issues/69892
+LINUX_NODES=$(kubectl get nodes -l beta.kubernetes.io/os=linux -o name)
+LINUX_NODE_COUNT=$(echo ${LINUX_NODES} | wc -w)
+for node in $LINUX_NODES; do
+  kubectl taint node $node node-under-test=false:NoSchedule
+done
+
+# Untaint the windows nodes to allow test workloads without tolerations to be
+# scheduled onto them.
+WINDOWS_NODES=$(kubectl get nodes -l beta.kubernetes.io/os=windows -o name)
+for node in $WINDOWS_NODES; do
+  kubectl taint node $node node.kubernetes.io/os:NoSchedule-
+done
+
+# Pre-pull all the test images. The images are currently hard-coded.
```

Comment:
Update this comment - the reference to PullImages.ps1 doesn't apply for the load testing, I don't think.